### ISSUE TITLE
Error: failed to add '*' to safe.directory

### ISSUE THREAD

**Issue by ivan-coder** [DeNA/unity-meta-check#28]
Hey, we've been using the action for months with no issues, but starting today we run into this error message:
```
Run DeNA/unity-meta-check@v3
  with:
    target_path: Survivor
    log_level: INFO
    target_type: auto-detect
    ignore_dangling: false
    ignore_case: false
    ignore_submodules_and_nested: false
    enable_autofix: false
    autofix_globs: *
    enable_junit: false
    enable_pr_comment: false
    pr_comment_lang: en
    pr_comment_pull_number: 0
    pr_comment_send_success: false
  
/usr/bin/docker run --name ghcriodenaunitymetacheckunitymetacheckghaction3[2](https://github.com/ivan-coder/survivor/actions/runs/8463559811/job/23192218534#step:4:2)1_216dd6 --label 1c4c[3](https://github.com/ivan-coder/survivor/actions/runs/8463559811/job/23192218534#step:4:3)a --workdir /github/workspace --rm -e "INPUT_TARGET_PATH" -e "INPUT_LOG_LEVEL" -e "INPUT_TARGET_TYPE" -e "INPUT_IGNORE_DANGLING" -e "INPUT_IGNORE_CASE" -e "INPUT_IGNORE_SUBMODULES_AND_NESTED" -e "INPUT_IGNORED_FILE_PATH" -e "INPUT_ENABLE_AUTOFIX" -e "INPUT_AUTOFIX_GLOBS" -e "INPUT_ENABLE_JUNIT" -e "INPUT_JUNIT_XML_PATH" -e "INPUT_ENABLE_PR_COMMENT" -e "INPUT_PR_COMMENT_LANG" -e "INPUT_PR_COMMENT_TMPL_FILE" -e "INPUT_PR_COMMENT_PULL_NUMBER" -e "INPUT_PR_COMMENT_SEND_SUCCESS" -e "HOME" -e "GITHUB_JOB" -e "GITHUB_REF" -e "GITHUB_SHA" -e "GITHUB_REPOSITORY" -e "GITHUB_REPOSITORY_OWNER" -e "GITHUB_REPOSITORY_OWNER_ID" -e "GITHUB_RUN_ID" -e "GITHUB_RUN_NUMBER" -e "GITHUB_RETENTION_DAYS" -e "GITHUB_RUN_ATTEMPT" -e "GITHUB_REPOSITORY_ID" -e "GITHUB_ACTOR_ID" -e "GITHUB_ACTOR" -e "GITHUB_TRIGGERING_ACTOR" -e "GITHUB_WORKFLOW" -e "GITHUB_HEAD_REF" -e "GITHUB_BASE_REF" -e "GITHUB_EVENT_NAME" -e "GITHUB_SERVER_URL"
WARNING: The requested image's platform (linux/arm6[4](https://github.com/ivan-coder/survivor/actions/runs/8463559811/job/23192218534#step:4:4)/v8) does not match the detected host platform (linux/amd[6](https://github.com/ivan-coder/survivor/actions/runs/8463559811/job/23192218534#step:4:6)4/v3) and no specific platform was requested
error: failed to add '*' to safe.directory
```

Our github action looks like this

```
name: Meta Check
on: push

jobs:
  meta-check:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: DeNA/unity-meta-check@v3
        with:
          target_path: Survivor
```
I saw the version bump a few hours ago, and figured it must be related. Any idea what can be happening?
Thanks!

**Comment by ivan-coder** [comment]
For anyone running into this issue, the previous version keeps running just fine:
`- uses: DeNA/unity-meta-check@v3.2.0`

**Comment by Kuniwak** [comment]
We are aware of the issue. We have just made a release on Apple Silicon Macs. Until now, we have been using Intel Macs, so the environment for the image has changed. We are currently working on this issue.

**Comment by Kuniwak** [comment]
@ivan-coder This problem was fixed at [v3.2.2](https://github.com/DeNA/unity-meta-check/releases/tag/v3.2.2). Please try it!

**Comment by ivan-coder** [comment]
Just to let you guys know, it fixed it indeed 👍
